README.md

```diff
@@ -16,20 +16,17 @@ MixedReality-WebRTC is a collection of libraries to help mixed reality app devel
 
-## Notice
+<h2><b><span style="color:red">Caution</span></b></h2>
 
-**With the announcement of Microsoft Mesh, we want to provide an update regarding our plans for the future of the MR-WebRTC project. Microsoft Mesh provides a fabric to enable collaborative scenarios for mixed reality that span across multiple platforms and endpoints and has additional features and capabilities beyond what is covered by the MR-WebRTC project. With that in mind, we have decided to scale back our development resources in the MR-WebRTC project and focus those resources into providing a great platform with Microsoft Mesh. For additional information about Microsoft Mesh, please check out our technical overview at [https://techcommunity.microsoft.com/t5/mixed-reality-blog/microsoft-mesh-a-technical-overview/ba-p/2176004](https://techcommunity.microsoft.com/t5/mixed-reality-blog/microsoft-mesh-a-technical-overview/ba-p/2176004).**
- 
-**While we are reducing our development investment in MR-WebRTC, we will continue to host it as a project, and accept pull-requests for the project. We also encourage anyone wishing to continue to develop the project to fork it and move forward with it.**
- 
-**If anyone has any questions, please feel free to reach out to Jesse McCulloch from our Developer Ecosystem team at [jemccull@microsoft.com](mailto:jemccull@microsoft.com).**
+<span style="color:red">**MR-WebRTC has been deprecated. We're no longer committing development resources to it, taking pull requests for it, or planning a replacement for it.**</span>
 
+- If you want to continue updating it, you must fork and maintain your own branch.
 
 ## Download
 
 NuGet (C++, C#) and UPM (Unity) packages are available for stable releases (`release/*` branches). See the [Release page on GitHub](https://github.com/microsoft/MixedReality-WebRTC/releases) or the [Download documentation page](https://microsoft.github.io/MixedReality-WebRTC/manual/download.html) for details.
 
-_Note_: The `master` branch contains the code for the next release, and therefore sometimes contains breaking API changes from the latest stable release. It is therefore not guaranteed to work with NuGet packages, which are only available for stable releases. In particular, the Unity library scripts are only guaranteed to be compatible with NuGet packages if copied from a `release/*` branch, though it is strongly recommended to use the UPM packages instead.
+_Note_: The `master` branch may contain breaking API changes from the latest stable release. It's  therefore not guaranteed to work with NuGet packages, which are only available for stable releases. In particular, the Unity library scripts are only guaranteed to be compatible with NuGet packages if copied from a `release/*` branch, though it is strongly recommended to use the UPM packages instead.
 
 ## Build Status

```

Comment:
Rename to "Branches" and delete build status table.

Reply:
Done.